src/Pages/archive/Archive.jsx

```diff
@@ -66,12 +68,12 @@ const editNoteHandler = (e) => {
         addFormFocus: false, 
     }});
 }
-
+const notesAfterFilterSort = getFilteredSortedNotes(archives, filterBylabels, sortBy, sortByPriority, searchText);
```

Comment:
Rather than importing sortfilterstate to pass it down to this getfilteredsortednotes you can make this a custom hook then important the sortfilterstate there only,